website/src/components/security-header/security-header.module.css

```
@@ -50,3 +50,14 @@
   border-bottom: 1px solid rgb(255, 255, 255, 0.2);
   width: 100%;
 }
+
+.link {
+  color: #fff;
+  font-weight: 600;
+  border-bottom: 2px solid rgba(255, 255, 255, 0.25);
+}
+
+.link:hover {
+  color: rgba(255, 255, 255, 0.7);
+  border-bottom: 2px solid rgba(255, 255, 255, 0.1);
```

You could just change the color since the rest is inherited from above?

```suggestion
  border-bottom-color: rgba(255, 255, 255, 0.1);
```